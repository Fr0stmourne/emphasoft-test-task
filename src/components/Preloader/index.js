import React from 'react'
import CircleLoader from 'react-spinners/CircleLoader'
import './index.scss'

export default function Preloader() {
  return (
    <div className="preloader">
      <CircleLoader size={'100%'}/>
    </div>
  )
}