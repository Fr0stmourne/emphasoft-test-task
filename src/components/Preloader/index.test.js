import React from 'react';
import renderer from 'react-test-renderer';
import Preloader from '.';

describe('Preloader component', () => {
  test('should render the component correctly when no props are provided', () => {
    const tree = renderer.create(<Preloader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
