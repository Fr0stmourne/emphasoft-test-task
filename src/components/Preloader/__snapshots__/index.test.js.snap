// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader component should render the component correctly when no props are provided 1`] = `
<div
  className="preloader"
>
  <div
    className="css-11linkp"
  >
    <div
      className="css-1goakdm"
    />
    <div
      className="css-odb1kq"
    />
    <div
      className="css-1viw8f6"
    />
    <div
      className="css-qnarmt"
    />
    <div
      className="css-1choe6w"
    />
  </div>
</div>
`;
